Composer/package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "resolutions": {
     "@types/react": "16.9.23",
-    "mkdirp": "^0.5.2"
+    "mkdirp": "^0.5.2",
+    "serialize-javascript": "^3.1.0"
```

Comment:
Why is this library added here. Isn't this a transitive dependacy fixed by bumping webpack?